### PR TITLE
Adding ascii table

### PR DESCRIPTION
**Pull request by r00tus3r**
#9699 Add command to show ascii table

### CODE REVIEW

**Comment by Maijin** [comment]
image

waaat

**Comment by radare** [comment]
the rtti thing is unrelated, but.. if it doesnt breaks maybe good to merge too

File: libr/anal/rtti.c [hunk 1]
```diff
@@ -32,12 +32,32 @@ static void rtti_msvc_print_all(RVTableContext *context, int mode) {
...
+	RListIter *vtableIter;
+	RVTableInfo *table;
+
+	if (vtables) {
+		r_list_foreach (vtables, vtableIter, table) {
+			if (r_cons_is_breaked ()) {
+				break;
+			}
+			r_anal_rtti_gcc_print_at_vtable (context, table->saddr, mode);
+			r_cons_print ("\n");
```
Comment (radare) [inline libr/anal/rtti.c]: you can use r_cons_newline() too

Comment (r00tus3r) [reply libr/anal/rtti.c]: Okay

**Comment by Maijin** [comment]
would be nice to have this within r2 too

**Comment by r00tus3r** [comment]
Where should I add it?

**Comment by radare** [comment]
I would say Under “p” to print.. but as long as its a reference and not printing actual data would make more sense under ?a or ?A

**Comment by r00tus3r** [comment]
Okay...I'll add it under ?a

**Comment by radare** [comment]
Thanks!

**Comment by radare** [comment]
I would put this string in r_util so u can reuse it from rax2 and r2. Its better to make a function that returns a const string and you define it as static

**Pull request merged**